thirdeye/thirdeye-frontend/app/pods/services/rootcause-aggregates-cache/service.js

```diff
@@ -28,7 +23,7 @@ export default Service.extend({
 
   init() {
     this._super(...arguments);
-    this.setProperties({aggregates: {}, context: {}, pending: new Set(), errors: new Set()});
+    this.setProperties({ aggregates: {}, context: {}, pending: new Set(), errors: new Set() });
```

Comment:
Why not give aggregates value of `{}` property level?

Reply:
This is a prettify of older code that I didn't write.  Are you thinking it needs to be refactored?

Reply:
we can refactor when we migrate code.

Reply:
I'll leave this one for later consideration.  It seems good for now.